src/services/etherpad/hooks/Group.js

```diff
@@ -0,0 +1,49 @@
+const { authenticate } = require('@feathersjs/authentication').hooks;
+const { disallow } = require('feathers-hooks-common');
+const { Forbidden } = require('@feathersjs/errors');
+
+const logger = require('../../../logger');
+
+const {
+	// getUser,
+	restrictToUsersOwnCourses,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['restrictToUsersOwnCourses' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5525245)